content/articles/2017-12-testthat-2-0.Rmd

```diff
@@ -0,0 +1,207 @@
+---
+title: testthat 2.0.0
+author: Hadley Wickham
+date: '2017-12-19'
+slug: testthat-2-0-0
+categories:
+  - package
+tags: [package, r-lib]
+photo:
+  url: https://unsplash.com/photos/Lu2pfy_8VKg
+  author: Cameron Kirby
+---
+
+```{r setup, include = FALSE}
+library(testthat)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are very excited to announce that testthat 2.0.0 is now available on CRAN! Testthat makes it easy to turn your existing informal tests into formal automated tests that you can rerun quickly and easily. Learn more at <http://r-pkgs.had.co.nz/tests.html>. Install the latest version with:
+
+```{r, eval = FALSE}
+install.packages("testthat")
+```
+
+testthat 2.0.0 is a massive release containing a bunch of new features. For the full details, see the [release notes](https://github.com/r-lib/testthat/releases/tag/v2.0.0), or read the hightlights below:
+
+* A new default reporter revamps the output to make better use of colour.
+
+* New setup and teardown tools provide rich ways of running code before and
+  after each test file, and before and after all tests.
+
+* New and improved expectations make it easier to test printed output and 
+  precisely test conditions (i.e. errors, warnings, and messages).
+
+* Quasiquotation support makes it easier to embed tests in for loops and
+  functions and still get useful failure messages.
+
+* Along with these new features we unforunately also had to make a few breaking 
+  changes. We describe the symptoms and how to work around the changes below.
+  
+In addition, as part of a general process to make tidyverse and r-lib packages available more widely, we are now checking testthat with R 3.1, 3.2, 3.3, 3.4, and 3.5.
+
+## New default reporter
+
+A new default reporter, `ReporterProgress`, revamps the output to make use of colour and reveal details of failures as they occur:
+
+
+If you prefer the previous version, you can turn back time by setting `option(testthat.default_reporter = "summary")`.
```

Comment:
I wonder if it is worth mentioning you can also use `devtools::test(reporter = "summary")`. I think in general people are not aware how to run the various reporters.